package.json

```diff
@@ -65,10 +65,12 @@
     "nodemailer": "~1.11.0",
     "nodemailer-smtp-transport": "~0.1.13",
     "request": "2.88.0",
+    "smpp": "0.3.1",
+    "swagger-jsdoc": "^3.4.0",
```

Comment:
^ should be changed to ~ in these dependencies

Reply:
Fixed in 51271cd82fd35f866a09fed1f4debe74380050a0